apps/backend/src/libs/modules/controller/libs/types/api-handler-options.type.ts

```diff
@@ -1,16 +1,20 @@
+import { type FastifyRequest } from "fastify";
+
 type APIHandlerOptions<
 	T extends DefaultApiHandlerOptions = DefaultApiHandlerOptions,
 > = {
 	body: T["body"];
 	params: T["params"];
 	query: T["query"];
-	user?: T["user"];
+	request: FastifyRequest;
```

Comment:
We don't need request here. You need to get file from the plugin, which already has a request